ndk-glue/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Unreleased
+
+- **Breaking:** Changed `main` macro attribute syntax
+
+# 0.1.0 (2020-04-22)
```

Comment:
I know there's not a precedent for this in winit/etc, but in the 0.1.0 section I tend to put `- Initial release` just so it looks less naked... it's no prob either way though!

Reply:
Good point, added 👍